tags/piracy.tag

```diff
@@ -5,6 +5,6 @@ type: text
 Cracked servers are not supported here and by setting your server as so, you will not receive support here any further.
```

Comment:
>  Cracked servers are not supported here and by setting your server as so, you will not receive support here any further. 

Offline mode != piracy, maybe make the working a bit softer (`Offline mode servers are often used for piracy, which Geyser does not condone. If you're using it locally for testing that's fine, but`...)